doc/src/tutorial/solvers.rst

```diff
@@ -142,9 +142,9 @@ In the ``solveset`` module, the non linear system of equations is solved using
    ``solve`` can be used for such cases:
 
    >>> solve([x**2 - y**2/exp(x)], [x, y], dict=True)
-   ⎡⎧      ⎛-y ⎞⎫  ⎧      ⎛y⎞⎫⎤
-   ⎢⎨x: 2⋅W⎜───⎟⎬, ⎨x: 2⋅W⎜─⎟⎬⎥
-   ⎣⎩      ⎝ 2 ⎠⎭  ⎩      ⎝2⎠⎭⎦
+   ⎡⎧         ____⎫  ⎧        ____⎫⎤
+   ⎢⎨        ╱  x ⎬  ⎨       ╱  x ⎬⎥
+   ⎣⎩y: -x⋅╲╱  ℯ  ⎭, ⎩y: x⋅╲╱  ℯ  ⎭⎦
```

Comment:
While I do think this is an improvement in the output of `solve` this change does not make sense in this docstring so a different example should be used here.

Reply:
With this PR, I don't think you will ever get a solution in terms of `LambertW` anymore since `y` (in this sort of equation) will have lower complexity than `x`. So the example can be removed.

Reply:
The example could be changed to solve for `x` only.

Reply:
doh...good point